Gist/Managers/Models/Message.swift

```diff
@@ -28,27 +30,45 @@ public class Message {
 
     public init(messageId: String) {
         self.queueId = nil
-        self.gistProperties = GistProperties(routeRule: nil, elementId: nil, campaignId: nil, position: .center)
+        self.gistProperties = GistProperties(routeRule: nil, elementId: nil, campaignId: nil, position: .center, persistent: false)
         self.messageId = messageId
     }
 
     init(queueId: String? = nil, messageId: String, properties: [String: Any]?) {
         self.queueId = queueId
-        self.gistProperties = GistProperties(routeRule: nil, elementId: nil, campaignId: nil, position: .center)
+        self.gistProperties = GistProperties(routeRule: nil, elementId: nil, campaignId: nil, position: .center, persistent: false)
         self.messageId = messageId
 
         if let properties = properties {
             self.properties = properties
-            if let gist = self.properties["gist"] as? [String: String] {
+            if let gist = self.properties["gist"] as? [String: Any] {
                 var messagePosition = MessagePosition.center
-                if let position = gist["position"], let positionValue = MessagePosition.init(rawValue: position) {
+                if let position = gist["position"], let positionValue = MessagePosition.init(rawValue: String(describing: position)) {
                     messagePosition = positionValue
                 }
+                var routeRule: String? = nil
+                if let routeRuleApple = gist["routeRuleApple"] {
```

Comment:
you can use `as? String`, this way you can just assign it without `String(describing:...`